packages/host/src/index.ts

```diff
@@ -2,25 +2,14 @@
 // Distributed under the terms of the Modified BSD License.
 import { Endpoint, Remote, windowEndpoint, wrap } from 'comlink';
 import { CommandRegistry } from '@lumino/commands';
-import { ReadonlyPartialJSONObject } from '@lumino/coreutils';
-
-// For demo/dev purposes
-declare global {
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  interface Window {
-    commandBridge: Remote<unknown>;
-  }
-}
 
 /**
  * A bridge to expose actions on JupyterLab commands.
  */
 export class CommandBridge {
-  constructor(iframeId: string, options: CommandBridge.IOptions) {
+  constructor(iframeId: string) {
```

Comment:
Looks like we may not need the `@lumino/commands` import and the `CommandBridge` namespace anymore?